includes/admin/class-wc-stripe-privacy.php

```
@@ -350,7 +351,8 @@ protected function maybe_handle_subscription( $order ) {
 
 		$subscription = current( wcs_get_subscriptions_for_order( $order->get_id() ) );
 
-		$stripe_source_id = $subscription->get_meta( '_stripe_source_id', true );
+		$order_helper     = WC_Stripe_Order_Helper::get_instance();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Order_Helper' in method 'maybe_handle_subscription'. (undefined)

(StaticAccess)